packages/mdc-button/_mixins.scss

```diff
@@ -279,7 +279,7 @@
   &:disabled {
     @include mdc-feature-targets($feat-color) {
       @include mdc-theme-prop(background-color, transparent);
-
+      fill: $mdc-button-disabled-ink-color;
```

Comment:
`fill` is only applicable for icon block elements, so it needs to be moved to `mdc-button__icon_` mixin.

Also, this can use `fill: currentColor` instead.